Resources/Prototypes/Voice/speech_emotes.yml

```diff
@@ -242,6 +242,31 @@
     - snapping fingers
     - snapped fingers
 
+- type: emote
+  id: Thump
+  name: chat-emote-name-thump
+  category: Hands
```

Comment:
Hey you might want to make new category! If its hands, stuff like #32588 might block it even if it shouldn't (E.g cuffs would block this even though that doesn't make sense.

Reply:
I'll worry about this if it gets merged